src/openrct2/actions/TrackRemoveAction.cpp

```diff
@@ -114,7 +114,7 @@ GameActions::Result TrackRemoveAction::Query() const
         return GameActions::Result(GameActions::Status::InvalidParameters, STR_RIDE_CONSTRUCTION_CANT_REMOVE_THIS, STR_NONE);
     }
 
-    if (tileElement->AsTrack()->IsIndestructible())
+    if (tileElement->AsTrack()->IsIndestructible() && !gCheatsMakeAllDestructible)
```

Comment:
Maybe the `isIndestructible` function itself should check the cheat?

Reply:
Yup makes sense, [80eb038](https://github.com/OpenRCT2/OpenRCT2/pull/19924/commits/80eb038c24e20cb9dc01e4e2a52b4374b4c2a429)